code/game/objects/items/food/burgers.dm

```diff
@@ -297,6 +297,14 @@
 	foodtypes = GRAIN | MEAT | DAIRY
 	venue_value = FOOD_PRICE_EXOTIC
 
+/obj/item/food/burger/superbite/suicide_act(mob/user, show_message=TRUE)
+	. = OXYLOSS
```

Comment:
Just `return OXYLOSS` in both paths, uses of `.` like this make code more confusing.